### DISCUSSION TITLE
Get data from two separate arrays of objects, both containing identical "key" names

### DISCUSSION THREAD

**Discussion by lilgrant22** [nlohmann/json#4021] [Q&A]
I need to get the data from this json message response:

```json
{
	"command": "get",
	"variables": {

		neighbors_json": {

			"value": [{

				"name": "5.1Node",
				"ip": "178.65.8.99"
				"speed": 87.0000002,
				"lat": 78.327247239,
				"lon": 45.23628,
				"alt": 143.0,
				"dist": 16.7788289928
			},{
				
				"name": "5.99Node",
				"ip": "178.65.8.72"
				"speed": 84.0440002,
				"lat": 71.32984247239,
				"lon": 52.287628,
				"alt": 148.0,
				"dist": 10.7787839328
			}]
		},

		"msgtype": "rep",

	},

	"token": "7kanavfy78",

}
```

The issue is that both of them I guess have the same key names (i.e. "name", "ip", "speed") and I need them to be stored to separate variables (i.e. i'll be setting the first name to variable name_5.1Node and the second name to name_5.99Node.

Is there a way to do this? Thanks

**Comment by nlohmann** [comment]
Yes, you can use `operator[]` with numbers for array indexing or strings for object indexing. See https://github.com/nlohmann/json#json-as-first-class-data-type
